### ISSUE TITLE
Shot Generator

### ISSUE THREAD

**Issue by DKKean** [wonderunit/storyboarder#2397]
There seems to be no place to type near, under or anywhere near the Shot Generator panel on my screen.  See attached, please.

![2022-12-05_12-17-07](https://user-images.githubusercontent.com/107891321/205700968-31ecf189-950e-4bee-a0be-a24c57c395d8.png)

**Comment by DKKean** [comment]
I'm referencing this 'getting started' text on the website:

Generate a shot just by typing...
Can't draw? Have a shot list? Or just want to quickly mockup reference shots without drawing? Just type a description into the Shot Generator in the sidebar, press return, and generate as many shots as you'd like. Change parameters as you want. Shots are fully customizable.

**Comment by aarcangeli** [comment]
[v1.9.2](https://github.com/wonderunit/storyboarder/releases/tag/v1.9.2) is the last version where this feature was present, I don't know why they removed it.

**Comment by ForceConstant** [comment]
> @aarcangeli 

Yeah I just downloaded this software to use this specific feature, and came here when I couldn't find it.. I do see some references in the code. 

https://github.com/wonderunit/storyboarder/blame/642e8f0df24a1b52744880f5bfa40d9e92e51d95/src/main-window.html#L454

**Comment by sebastian1111** [comment]
Can that code be somehow added back ? That function is quite useful. For example just try to write in the old version: "man, looking up" . And you already have a man with his head tilted up and in an interesting framing. If you try to do this manually in the newer version it will  take you much longer and fumbling with all sort of controls.

The ideal way would be probably to have this fancy "prompt" thing mixed in with the shot generator. The prompt could be located  inside, after you enter the Shot Generator window. So you can change the angle of view and other properties after you wrote the lines which generated the characters and the scene.
